cpp/src/holtwinters/runner.h

```diff
@@ -30,10 +30,10 @@ void HWTranspose(const ML::cumlHandle &handle, Dtype *data_in, int m, int n,
                  Dtype *data_out) {
   ASSERT(!(!data_in || !data_out || n < 1 || m < 1), "HW error in in line %d",
          __LINE__);
-  const ML::cumlHandle_impl &handle_impl = handle.getImpl();
+  const auto &handle_impl = handle.getImpl();
```

Comment:
Why this change as part of this PR. IMHO it is better to keep PRs minimal. Applies to other `auto` below as well.